src/pages/ReportSettingsPage.js

```diff
@@ -128,26 +192,22 @@ class ReportSettingsPage extends Component {
                         <View style={[styles.flexRow]}>
                             <View style={[styles.flex3]}>
                                 <RoomNameInput
-                                    onChangeText={newRoomName => this.setState({newRoomName})}
-                                    onChangeError={error => this.setState({error})}
                                     initialValue={this.state.newRoomName}
-                                    disabled={shouldDisableRename}
                                     policyID={linkedWorkspace && linkedWorkspace.id}
+                                    errorText={this.state.errors.newRoomName}
+                                    onChangeText={newRoomName => this.clearErrorAndSetValue('newRoomName', newRoomName)}
+                                    disabled={shouldDisableRename}
                                 />
                             </View>
                             <Button
                                 success={!shouldDisableRename}
                                 text={this.props.translate('common.save')}
-                                onPress={() => Report.renameReport(this.props.report.reportID, this.state.newRoomName)}
+                                onPress={this.validateAndRenameReport}
                                 style={[styles.ml2, styles.flex1]}
                                 textStyles={[styles.label]}
                                 innerStyles={[styles.reportSettingsChangeNameButton]}
-                                isDisabled={Boolean(
-                                    shouldDisableRename
-                                    || this.state.newRoomName === this.props.report.reportName
-                                    || this.state.error,
-                                )}
                                 isLoading={this.props.isLoadingRenamePolicyRoom}
+                                isDisabled={shouldDisableRename}
```

Comment:
NAB (as we are conducting an audit of all forms now) but no form buttons should be disabled

Reply:
I think this needs to be disabled for this case when user is seeing settings for default policies #admins, etc. I haven't seen the new Form style draft so I can't think of anything apart from it.

Reply:
We'll release the guidelines soon. Basically, no forms should be disabled for any reason.

Reply:
Ok. Thanks.